ports/simbody/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "simbody",
-  "version-date": "2022-01-04",
-  "port-version": 2,
+  "version-date": "2023-01-10",
+  "port-version": 3,
```

```suggestion
  "version-date": "2023-01-10",
```

When the `version-date` changes, the `port-version` should be restored to 0 (when it is 0, the `port-version` is generally deleted).